js/foundation.reveal.js

```diff
@@ -406,7 +410,7 @@ class Reveal extends Plugin {
 
       _this.$element.attr('aria-hidden', true);
 
-      _this._enableScroll();
+      _this._enableScroll(scrollTop);
```

Comment:
Could we move `enableScroll` before removing `is-reveal-open`, instead of passing the scrollTop ? I think this would be cleaner, but I don't know the codebase enough to be sure this would not cause others bugs.

Reply:
I first tried this way and it fixed the iOS issue (original scrollTop restored after having closed the reveal).
But unfortunately non-iOS / desktop has got the issue then!

The reason is that the class `is-reveal-open` adds fixed position to the html and thus the scrollTop restoration don't work before removing that class.

My solution was the only way I've found to get rid of the issue for all cases.

Reply:
@ncoden what do you think?

Would be great to see this iOS issue be gone in my projects (without adjusting the sources temporary)

Reply:
Sorry for the late reply. I'll check if it can be cleaner, then merge this.